Formula/kitchen-sync.rb

```diff
@@ -13,17 +13,19 @@ class KitchenSync < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "pkg-config" => :build
-  depends_on "boost"
+  depends_on "libpq"
   depends_on "mysql-client"
-  depends_on "yaml-cpp"
 
-  needs :cxx11
+  needs :cxx14
 
   def install
     ENV.cxx11
```

Comment:
Could you update this to `ENV.cxx14` as well ?

Reply:
There doesn't seem to be such a method in Homebrew.

(In fact, I don't know why the ENV.cxx11 call is there in the original recipe, because the project's CMakefile adds the necessary compiler flags for C++14 itself.  But it wasn't related to what I was fixing, so I left it.)

Reply:
Here's the cxx11 method definitions I can find:
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/ENV/std.rb#L168
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/ENV/super.rb#L303
No cxx14 method...

Reply:
Builds fine from source without `ENV.cxx11`.  Think I should remove it while I'm here?

Reply:
Yes. `ENV.cxx11` was possibly made necessary by a previous `boost` dependency.